### PR TITLE
Make summarize in export block properly

### PR DESCRIPTION
**Pull request by dominiklohmann**
Note that this fix is for `export` only, this will stay slightly incorrect for both partition transformations and `import` until we have the new executor in place.

### CODE REVIEW

File: libvast/src/system/exporter.cpp [hunk 1]
```diff
@@ -232,15 +237,15 @@ exporter(exporter_actor::stateful_pointer<exporter_state> self, expression expr,
         : query_context::priority::normal;
   VAST_DEBUG("spawned exporter with {} pipelines", pipelines.size());
   self->state.pipeline = pipeline_executor{std::move(pipelines)};
```
Comment (tobim) [inline libvast/src/system/exporter.cpp:239]: Doesn't make sense to get results in batches in this case.
```suggestion
  self->state.pipeline = pipeline_executor{std::move(pipelines)};
  if (self->state.pipeline.is_blocking())
    self->state.query_context.taste = std::numeric_limits<uint32_t>::max();
```

File: libvast/include/vast/pipeline.hpp [hunk 2]
```diff
@@ -39,7 +39,7 @@ class pipeline {
   void add_operator(std::unique_ptr<pipeline_operator> op);
 
   /// Returns true if any of the pipeline operators is aggregate.
-  [[nodiscard]] bool is_aggregate() const;
+  [[nodiscard]] bool is_blocking() const;
```
Comment (tobim) [inline libvast/include/vast/pipeline.hpp:42]: The comment needs to be adjusted.

**Review by tobim — APPROVED** [review]
Thanks for the fix!

File: libvast/src/system/exporter.cpp (hunk 1 above)
Comment (dominiklohmann) [reply libvast/src/system/exporter.cpp:239]: Nice catch!

**Pull request merged**